app/models/bookings/school.rb

```diff
@@ -139,6 +139,10 @@ def notifications_email
     end
   end
 
+  def admin_contact_name
+    profile&.admin_contact_full_name
```

Comment:
Surely this wouldn't handle schools which hadn't onboarded? So wouldn't have a profile? I think this should be

```ruby
profile ? profile.admin_contact_full_name : contact_email
```

Reply:
We only use this method in the cancellation emails we send to the school. I've spoken to Fred about this and we've decided to remove the admin name from the emails and replace it with just 'Hello'. I'll shoot up a pr to remove this method as it's redundant .

Reply:
Disregard above, I'll issue a pr to fall back to the contact email like you suggested!

Reply:
Fixed the issue here https://github.com/DFE-Digital/schools-experience/pull/530